docs/doc/10-deploy/09-upgrade/90-upgrade-1.0.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Upgrade Databend to 1.0
+sidebar_label: Upgrade Databend to 1.0
+description:
+    Uprgade Databend to 1.0 from 0.8 or later
+---
+
+This topic explains how to upgrade Databend to version 1.0 from a version 0.8 or later.
+
+## Before You Start
+
+Before upgrading, make sure you have completed the following tasks:
+
+- Read and understand the [General Principles](50-upgrade.md#general-principles) about upgrading Databend.
+
+- Back up your Meta data. For how to back up and restore Meta data, see [Back Up and Restore Databend Meta Service Cluster](../08-backup-restore/10-backup-and-restore-schema.md).
+
+- Go to the [Download](https://databend.rs/download) page and download the right installation package for your platform.
+
+## Step 1: Upgrade databend-query
+
+Upgrade the databend-query service cluster by cluster. Follow the steps below to upgrade the databend-query service in each node:
+
+1. Shut down the running databend-query service.
+
+```shell
+killall databend-query
+```
+
+2. Replace the binary file `databend-query` in the folder where Databend is installed with the newer version.
+
+3. Remove the following deprecated settings from your configuration file `databend-query.toml`:
+
+```toml
+database_engine_github_enabled = true
+table_cache_enabled = true
+table_memory_cache_mb_size = 1024
+table_disk_cache_root = "_cache"
+table_disk_cache_mb_size = 10240
+table_cache_bloom_index_meta_count=3000
+table_cache_bloom_index_data_bytes=1073741824
+```
+
+4. Add the following new settings to the `[query]` section in your configuration file `databend-query.toml`:
+
+```toml
+default_storage_format = 'parquet'
+default_compression = 'zstd'
+```
+
+5. Add the following cache settings to the end of your configuration file `databend-query.toml`:
+
+```toml
+# Cache config.
+[cache]
+# Type of storage to keep the table data cache
+#
+# available options: [none|disk]
+# default is "none", which disable table data cache
+# use "disk" to enabled disk cache
+data_cache_storage = "none"
+
+[cache.disk]
+# cache path
+path = "./.databend/_cache"
+# max bytes of cached data 20G
+max_bytes = 21474836480
+```
+
+6. Start the new databend-query service.
+
+```shell
+cp ~/tmp/bin/databend-query bin/
+./bin/databend-query --version # Confirm the version number
+ulimit -n 65535
+nohup bin/databend-query --config-file=configs/databend-query.toml 2>&1 >query.log &
+```
+
+## Step 2: Upgrade databend-meta
+
+To upgrade the databend-meta service, you need to use the binary file `databend-meta-upgrade-09`, which can be located in the `bin` folder of your current Databend installation directory.
```

Comment:
It would be better to add the explanation:
databend-meta `v0.8` ->`v 0.9|1.0` should use the `databend-meta-upgrade-09`, this is a breaking change, `v0.9->v1.0` should not use this upgrade tool.

Reply:
v1.0 databend-meta ver: 30 , but v0.9 from 25-29, so need upgrade it , if you want use databend-meta v1 replace databend-meta v0.9.

Reply:
https://github.com/datafuselabs/databend/blob/b7c24c1a836f6ba6b474452ef8deeb94464e8ab1/src/meta/proto-conv/src/util.rs#L30-L96